src/components/product_menu_components.rs

```diff
@@ -0,0 +1,89 @@
+use maud::{html, Markup};
+
+use crate::dso::product::Product;
+
+pub fn products_table_component(products: &[Product]) -> Markup {
+    let left_products = products.iter().step_by(2);
+    let right_products = products.iter().skip(1).step_by(2);
+
+    products_table_template_component(
+        products_table_part_component(left_products),
+        products_table_part_component(right_products),
+    )
+}
+
+pub fn clickable_products_table_component(products: &[Product], username: &str) -> Markup {
+    let left_products = products.iter().step_by(2);
+    let right_products = products.iter().skip(1).step_by(2);
+
+    products_table_template_component(
+        clickable_products_table_part_component(left_products, username),
+        clickable_products_table_part_component(right_products, username),
+    )
+}
+
+// TODO: SHIT FUCKING NAME
```

Comment:
Wanna fix later or now?

Reply:
Later I wanna figure out the best way to do components when we have more of them